extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -110,6 +110,7 @@ export class AmpSidebar extends AMP.BaseElement {
     const toolbarElements =
     Array.prototype.slice
       .call(this.element.querySelectorAll('nav[toolbar]'), 0);
+    toolbarElements.reverse();
```

Comment:
you should not add it as the first child , you will need to add it as the last child so that you dont break existing css. The order of insertion of toolbars should not matter